webapp/app/[locale]/stake/_components/rewardTag.tsx

```diff
@@ -0,0 +1,63 @@
+import { DiamondPointsIcon } from '../../stake/_components/icons/diamondPoints'
+import { HemiPointsIcon } from '../../stake/_components/icons/hemiPoints'
+import { PointsIcon } from '../../stake/_components/icons/points'
+import { SolvXpPointsIcon } from '../../stake/_components/icons/solvXpPoints'
+
+type Props = {
+  label: string
+  icon: React.ReactNode
+  style?: React.CSSProperties
+}
+
+const RewardTag = ({
+  label,
+  icon,
+  backgroundColor = 'bg-orange-500',
+  style,
+}: {
+  backgroundColor?: 'bg-orange-500' | 'bg-purple-900/70' | 'bg-neutral-950'
+} & Pick<Props, 'style' | 'label' | 'icon'>) => (
```

Comment:
It seems to me here it got mixed up? There's only one component using this Props, perhaps you want to merge them?

```suggestion
type Props = {
  backgroundColor?: 'bg-orange-500' | 'bg-purple-900/70' | 'bg-neutral-950'
  label: string
  icon: React.ReactNode
  style?: React.CSSProperties
}

const RewardTag = ({
  label,
  icon,
  backgroundColor = 'bg-orange-500',
  style,
}:  Props) => (
```

Reply:
Yes this also works. I followed your earlier suggestion.